FitGoal/FitGoal/Singup&Singin/CustomAuthenticator.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  CustomAuthenticator.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 10/4/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import Foundation
+
+class CustomAuthenticator {
+    var userName: UserInfoField?
+    var userEmail: UserInfoField?
+    var password: UserInfoField?
+    var passwordConfirmation: UserInfoField?
```

Comment:
Not a big fan of having all these optionals here. I think that what should be `nil` is their associated values instead.